nurseconnect/constants.py

```diff
@@ -0,0 +1,10 @@
+from django.utils.translation import ugettext_lazy as _
+
+MALE = "m"
+FEMALE = "f"
+UNSPECIFIED = "-"
+GENDERS = {
```

Comment:
Sure, i'll change it to `SEX`, but with regards to data above comment, the constants.py file was formatted with the same structure as all molo constants.py files, so shouldn't we rather keep to that style of formatting?